packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -1043,6 +1049,28 @@ enum NullSafetyMode {
   autodetect,
 }
 
+/// Indicates the module system DDC is targeting.
+enum DdcModuleFormat {
+  amd,
+  ddc,
+  es6,
```

Comment:
Why are we adding support for es6?

Reply:
That was purely for "completeness". Removed, as we don't plan on extending ES6 support.